README.md

```diff
@@ -1,6 +1,19 @@
-# Getting Started with Create React App
+## Project Description
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A website in which customers can reserve rockets and mission of space-X
+
+### **Student A**
+
+Chuanqi Zhu
+
+### **Student B**
+
+Stephenia Ugboaja Uchechi
```

Comment:
[ Optional] Why don't we add your social links, this helps recruiters to get in touch after checking out your project 😉

Reply:
Thanks for the suggestion. Social links have been added.